app/components/node/connect/node-connect.html

```diff
@@ -1,37 +1,75 @@
 <div class="slideout-form content-wrapper node-connect" *ngIf="node && pool">
     <div class="connect-content">
-        <h1>Connect to node </h1>
-        <div *ngIf="credentialSource === null">
-            <h3>Node credentials</h3>
+        <div class="sidebar-header">
+            <h1>Connect to node</h1>
+        </div>
+        <div *ngIf="!formVisible">
+            <div class="node-information">
+                <bl-property-group>
+                    <!-- If waiting for connection settings -->
+                    <div *ngIf="!connectionSettings && !ipFromRDP">
```

Comment:
This file is starting to get quite big, Might be worth to see if we can create sub components(Like the property display might be able to be a separate component)